mutiny-core/src/peermanager.rs

```diff
@@ -323,7 +323,7 @@ pub(crate) async fn connect_peer_if_necessary<S: MutinyStorage>(
         // first check to see if the fee rate is mostly up to date
         // if not, we need to have updated fees or force closures
         // could occur due to UpdateFee message conflicts.
-        fee_estimator.update_fee_estimates_if_necessary().await?;
+        // fee_estimator.update_fee_estimates_if_necessary().await?;
```

Comment:
I'm guessing this is just from testing, should be able to bring back. Maybe just put this behind `if self.network != Network::Regtest` this could be a problem for other people too.